src/modelinstance.cpp

```diff
@@ -691,6 +688,17 @@ Status ModelInstance::loadOVCompiledModel(const ModelConfig& config) {
         return status;
     }
 
+    uint32_t numberOfStreams = 0;
+    try {
+        numberOfStreams = getNumOfStreams();
+        SPDLOG_INFO("Number of OpenVINO streams: {}", numberOfStreams);
+    } catch (ov::Exception& e) {
+        SPDLOG_WARN("Unable to get information about number of streams with error: {}", e.what());
+    } catch (...) {
+        SPDLOG_WARN("Unable to get information about number of streams");
```

Comment:
```suggestion
        SPDLOG_LOGGER_INFO(modelmanager_logger,"Number of OpenVINO streams: {}", numberOfStreams);
    } catch (ov::Exception& e) {
        
    } catch (...) {
        SPDLOG_LOGGER_WARN(modelmanager_logger,"Unable to get information about number of streams for model: {}; version: {}; device: {}; with error: {}", getName(), getVersion(), config.getTargetDevice(), e.what());
```

Reply:
1) Use appropiate logger
2) I think we should add info which model is it to the warn since someone can just use WARN log level.

With INFO we are probably fine to skip model name/version since model loading is single thread.

Reply:
Done